src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -1549,6 +1554,9 @@ def generate_snapshot_batch(self, batch_results: pd.DataFrame, rom_name: str, na
             columns = batch_results.columns[1::]
             outpath = []
```

Comment:
Now that we're outputting either paths, or arrays, I'd suggest renaming 'outpath' to something like 'snapshots' or 'outputs' to remove ambiguity

Reply:
I have included these changes, thanks